package.json

```diff
@@ -59,6 +59,7 @@
     "IE 11"
   ],
   "dependencies": {
+    "@vaadin/vaadin-usage-statistics": "^2.0.0-alpha4",
```

Comment:
The PR title states "in the dev mode", but `@vaadin/vaadin-usage-statistics` is in `dependencies`.

Expected: `@vaadin/vaadin-usage-statistics` is in `devDependencies`.